Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst

```diff
@@ -19,92 +19,98 @@ module while TypoScript frontend plugin can only be added via TypoScript or
 
-   The action :php:`delete` of the :php:`CommentController` is not listed. This
-   action is therefore not allowed in this plugin.
+    The action :php:`delete` of the :php:`CommentController` is not listed. This
+    action is therefore not allowed in this plugin.
 
-   The TypoScript of the plugin will be available at
-   :typoscript:`tt_content.list.20.blogexample_postsingle`. Additionally
-   the lists of allowed and non-cacheable actions have been added to the
-   according global variables.
+    The TypoScript of the plugin will be available at
+    :typoscript:`tt_content.list.20.blogexample_postsingle`. Additionally
+    the lists of allowed and non-cacheable actions have been added to the
+    according global variables.
 
-#. :php:`registerPlugin()`: Add to :sql:`list_type` :sql:`tt_content`.
+#.  :php:`registerPlugin()`: Add to :sql:`list_type` :sql:`tt_content`.
 
-   Make the plugin available in the field
-   :guilabel:`Plugin > Selected Plugin`, :sql:`list_type` of the table
-   :sql:`tt_content`.
+    Make the plugin available in the field
+    :guilabel:`Plugin > Selected Plugin`, :sql:`list_type` of the table
+    :sql:`tt_content`.
 
-   .. figure:: /Images/ManualScreenshots/Extbase/ListType.png
-      :class: with-shadow
+    ..  figure:: /Images/ManualScreenshots/Extbase/ListType.png
+        :class: with-shadow
 
-      The new plugin in the content record at :guilabel:`Plugin > Selected Plugin`
+    The new plugin in the content record at :guilabel:`Plugin > Selected Plugin`
 
-   .. include:: /CodeSnippets/Extbase/FrontendPlugins/RegisterPlugin.rst.txt
+    ..  literalinclude::  _FrontendPlugin/_tt_content.php
+        :language: php
+        :caption: EXT:blog_example/Configuration/TCA/Overrides/tt_content.php
 
    Use the following parameters:
 
-   #. Extension key :php:`'blog_example'` or name :php:`BlogExample`.
-   #. A unique identifier for your plugin in UpperCamelCase: :php:`'PostSingle'`,
-      must be the same as used in :php:`configurePlugin()` or the plugin will
-      not render.
-   #. Plugin title in the backend: Can be a string or a localized string starting
-      with :php:`LLL:`.
-   #. (Optional) the :ref:`icon identifier <icon>` or file path prepended with "EXT:"
+    #.  Extension key :php:`'blog_example'` or name :php:`BlogExample`.
+    #.  A unique identifier for your plugin in UpperCamelCase: :php:`'PostSingle'`,
+        must be the same as used in :php:`configurePlugin()` or the plugin will
+        not render.
+    #.  Plugin title in the backend: Can be a string or a localized string starting
+        with :php:`LLL:`.
+    #.  (Optional) the :ref:`icon identifier <icon>` or file path prepended with "EXT:"
 
-#. (Optional) Add to the :guilabel:`New Content Element` wizard
+#.  (Optional) Add to the :guilabel:`New Content Element` wizard
 
-   Add the following page TSconfig to add the new plugin to the wizard:
+    Add the following page TSconfig to add the new plugin to the wizard:
```

Comment:
Can we link page TSconfig to the reference? New users might not know what that is.